hpat/datatypes/hpat_pandas_series_functions.py

```diff
@@ -40,6 +40,34 @@
 
 from hpat.hiframes.pd_series_ext import SeriesType
 
+'''
+Pandas Series (https://pandas.pydata.org/pandas-docs/stable/reference/series.html)
```

Comment:
Sorry but we changed documentation style. This section is not presented anymore here.